server/routes/api/profiles.js

```diff
@@ -9,6 +9,6 @@ const { newProfileValidator } = profileValidator;
 const { editProfile, getProfile } = ProfileController;
 
 router.put('/profiles', verifyToken, newProfileValidator, editProfile);
-router.get('/profiles/', verifyToken, getProfile);
+router.get('/profiles/:userId', verifyToken, getProfile);
```

Comment:
I thought the purpose of this PR is to get rid of the `userId` query params?

Reply:
Yes. I realized we need the userId after I was told by the team that we need users to be able to view other user's profile